adduser.js

```diff
@@ -0,0 +1,102 @@
+
+var name, fname, dob, email, username, password, admin;
+
+$(document).ready(function() {
+	// body...
+	$('#errorc2').hide();
+	$('#errorc3').hide();
+	$('#errorc4').hide();
+	$('#errorc5').hide();
+	$('#errorc6').hide();
+	$('#errorc7').hide();
+	$('#errorc8').hide();
+	$('#errorc9').hide();
+	$('#prev').prop('disabled', true);
+	$("#ok").prop('disabled', true);
+	
+	$('#addbutton').click(function(){
+		addUser();
+	});
+
+	$('#next').click(function(){
+		validateGeneral();
+	});
+
+	$('#nextcredential').click(function(){
+		validateCredential();
+	});
+
+	$('#okpermission').click(function(){
+		validatePermission();
+	});
+});
+
+function addUser(){
```

Comment:
Put every thing ready. And define function before you call other wise it will create problem in JS.